src/types/markers.js

```diff
@@ -145,6 +145,10 @@ export type MarkerSchema = {|
         label?: string,
         format: MarkerFormatType,
         searchable?: boolean,
+        // Optionally some data fields can be hidden in the list of fields in
+        // the tooltip or in the sidebar. But it can still be used inside labels
+        // and for searching.
+        hidden?: boolean,
```

Comment:
You use "fields" plural in the first sentence and then say "it" in the second sentence :)

I'd suggest the following:

> If present and set to true, this field will not be shown in the list of fields in the tooltip
> or in the sidebar. Such fields can still be used inside labels and they can be searchable.

Reply:
Ah welp, I tend to change my mind or forget what I wrote half way through when I'm typing something :D Changed it.